lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -222,6 +224,19 @@ defmodule ExUnit.Formatter do
     code_multiline(Macro.to_string(expr), padding_size)
   end
 
+  defp format_binding(binding, width) do
+    padding = @counter_padding <> "  "
+    padding_size = byte_size(padding)
+
+    result =
+      Enum.map_join(binding, "\n" <> padding, fn {var, value} ->
+        first_part = "#{var} = "
+        first_part <> inspect_multiline(value, padding_size + String.length(first_part), width)
```

Comment:
I don't think we should do `+ String.length`. For long variable names we will have very little space. I think indenting two spaces past the variable name will be fine.